CHANGELOG.md

```diff
@@ -3,7 +3,19 @@
 The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK
 also includes support for offline operations.
 
-## Unreleased
+## 3.1.4
+
+### Bug Fixes
+
+- Updated AppSync RealTime Client, which fixes stale connection handling upon resume from background:
+  - https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/393
```

Comment:
I do not believe that your change has a fix for: https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/393

Reply:
Sorry, was a copy/paste from a "refer to issue..." on the RTC project. Fixing